rllib/BUILD

```diff
@@ -1434,26 +1434,26 @@ py_test(
 )
 
 py_test(
-    name = "examples/parametric_action_cartpole_pg", main="examples/parametric_action_cartpole.py",
+    name = "examples/parametric_action_cartpole_pg", main="examples/parametric_actions_cartpole.py",
```

Comment:
Make sure to update the `doc` references to the files too, e.g., for this one
```
source/rllib-models.rst
277:Depending on your use case it may make sense to use just the masking, just action embeddings, or both. For a runnable example of this in code, check out `parametric_action_cartpole.py <https://github.com/ray-project/ray/blob/master/rllib/examples/parametric_action_cartpole.py>`__. Note that since masking introduces ``tf.float32.min`` values into the model output, this technique might not work with all algorithm options. For example, algorithms might crash if they incorrectly process the ``tf.float32.min`` values. The cartpole example has working configurations for DQN (must set ``hiddens=[]``), PPO (must disable running mean and set ``vf_share_layers=True``), and several other algorithms. Not all algorithms support parametric actions; see the `feature compatibility matrix <rllib-env.html#feature-compatibility-matrix>`__.

source/rllib-examples.rst
51:- `Parametric actions <https://github.com/ray-project/ray/blob/master/rllib/examples/parametric_action_cartpole.py>`__:
```

Reply:
done.